server/routes/index.ts

```diff
@@ -27,7 +28,11 @@ export default function routes(services: Services): Router {
   const urlTemplates = services.routeUrls.templates
 
   get(urlTemplates.home, (req, res) => {
-    res.render('pages/index')
+    if (process.env.NO_HOME_REDIRECT) {
+      res.send('')
+      return
+    }
```

Comment:
this is only for integration testing. there are no pages left on the site that don't require complicated stubbing and login/auth/token verification tests need _some_ simple page to visit to trigger sign-in processes